security-force-monitor/sfm-cms
# Adding cancel buttons when adding sources

This PR adds in "Cancel" buttons to the paths requested in #578 and #579. It also upgrades font-awesome to the most current version, because the old version was blocking me from adding another icon.

## Testing
- Go to a page in the tab you'll use to test
- Go to ~/en/create/source
- Click "Cancel"
- Confirm you are taken back to previous page
- Repeat steps when editing an access point of a source.
